generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management.tsx.ejs

```diff
@@ -122,13 +129,15 @@ export class UserManagement extends React.Component<IUserManagementProps, IPagin
                   <td>{user.login}</td>
                   <td>{user.email}</td>
                   <td>
-                    {
-                      user.activated ? (
-                        <span className="badge badge-success" style={{ cursor: 'pointer' }}>Activated</span>
-                      ) : (
-                        <span className="badge badge-danger" style={{ cursor: 'pointer' }}>Deactivated</span>
-                      )
-                    }
+                    {user.activated ? (
+                      <Button color="success" onClick={this.toggleActive.bind(this, { user }) }>
```

Comment:
dont bind use closure. And why is user passed inside another object, just pass directly